android/app/src/main/java/org/fossasia/openevent/status/BookmarkStatus.java

```diff
@@ -0,0 +1,18 @@
+package org.fossasia.openevent.status;
```

Comment:
A new package is not needed, just add it in the same package as its listener. The coupling between components should be reflected by the packaging